controllers/model/model_controller.go

```diff
@@ -210,7 +210,7 @@ func (r *ModelReconciler) initializeContext(ctx *reconcileRequestContext) error
 	ctx.SageMakerClient = clientwrapper.NewSageMakerClientWrapper(r.createSageMakerClient(awsConfig))
 	ctx.Log.Info("Loaded AWS config")
 
-	if ctx.Model.Spec.PrimaryContainer.Mode == nil {
+	if ctx.Model.Spec.PrimaryContainer != nil && ctx.Model.Spec.PrimaryContainer.Mode == nil {
```

Comment:
these changes have already been merged in a previous [PR](https://github.com/aws/amazon-sagemaker-operator-for-k8s/commit/c78903ced42b6a251e4f9bb3347846ce72bbbf71). Any idea why are these showing up?

same for controllers/sdkutil/spec_sdk_converters.go

Reply:
Oh odd. I mustn't be rebased maybe